packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -392,6 +394,7 @@ export class CliWrapper {
         target: string,
         authProvider: AuthProvider,
         workspaceFolder: Uri,
+        pythonExtension: MsPythonExtensionWrapper,
```

Comment:
Is the problem with the `deploy` only? Should we propagate the correct PATH to other bundle commands (like `run`)?

Reply:
Yeah, it is deploy only for now.